### ISSUE TITLE
small target detection no detections

### ISSUE THREAD

**Issue by JimmyHu1107** [tensorflow/models#10641]
# Prerequisites

Please answer the following question for yourself before submitting an issue.

- [ ] I checked to make sure that this feature has not been requested already.

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/blob/master/research/object_detection/configs/tf2/faster_rcnn_resnet50_v1_640x640_coco17_tpu-8.config

## 2. Describe the feature you request

I want to use the the config above and my small target detection dataset to gain weight,after first train ,i think maybe  the scale , hiight stride and width stride in  grid_anchor_generator are so big that the network can't get valid feature,and there are no detections in the inference

>      grid_anchor_generator {
>         scales: [0.25, 0.5, 1.0, 2.0]
>         aspect_ratios: [0.5, 1.0, 2.0]
>         height_stride: 16
>         width_stride: 16

after analyzing the dataset, i get the smallest scale,so I change to scales to :

>       grid_anchor_generator {
>         scales: [0.0078125,0.015625,0.03125,0.0625,0.125, 0.25, 0.5, 1.0]
>         aspect_ratios: [0.5, 1.0, 2.0]
>         height_stride: 2
>         width_stride: 2

but the the loss of the network is like this 
![image](https://user-images.githubusercontent.com/89967988/168780123-2bf4e776-f021-463c-aea9-52f8c75a54bf.png)
 i can't find the astringent point.and tno detection as above.
anyone has done like this,does the od api exist a model to suit the small target detection ,please give me some advice ,tks.
## 3. Additional context
one of my dataset like this:
![image](https://user-images.githubusercontent.com/89967988/168782297-22e9ab2a-7cfb-4354-90f1-d4676e6e3602.png)

**Comment by Petros626** [comment]
I would recommend you to use the tutorial: https://github.com/tensorflow/models/blob/master/research/object_detection/colab_tutorials/generate_ssd_anchor_box_aspect_ratios_using_k_means_clustering.ipynb. You will get float values, which match the best for your annotations.
